Baekjoon/10866.py

```diff
@@ -0,0 +1,54 @@
+'''
+문제 링크:https://www.acmicpc.net/problem/10866
+카테고리:덱
+문제해설:덱(Deque)를 구현한 다음, 입력으로 주어지는 명령을 처리
+'''
+from collections import deque
+import sys
+
+n=int(input())
+input=sys.stdin.readline
+
+d=deque([])
+
+for i in range(n):
+  commend=input().split()
+
+  if commend=='push_front':
+    d.appendleft()
+
+  if commend=='push_back':
+    d.append()
+
```

Comment:
`append()`, `appendleft()`함수는 args를 반드시 1개 이상 받아야 하니 위에 입력으로 받은 `commends`의 첫 번째 요소 `commends[1]`을 넣어줘야 해요.

`commends[1]`이 out of index error 가 발생하지 않도록 예외처리 해줘도 좋겠습니다.

```suggestion
  if commend=='push_front':
    d.appendleft(commends[1])

  if commend=='push_back':
    d.append(commends[1])

```

Reply:
args!!